### DISCUSSION TITLE
Add node.js

### DISCUSSION THREAD

**Discussion by jhon-cena77** [github-linguist/linguist#6507] [General]
I think having node.js separate from JavaScript would be good because node.js is in .mjs and .cjs but also .js you could detect functions that are only in node.js

**Comment by Nixinova** [comment]
Node.js is an environment that uses the language, JavaScript. Linguist is about recognizing only languages. For instance even React JSX is labelled JavaScript even though it is syntactically different from vanilla JS, its language is still just JavaScript. So no, Node.js won't be added as a separate language, because it isn't one.
